.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - uses: yusancky/setup-typst@v2
         id: setup-typst
         with:
-          version: 'v0.10.0'
+          version: 'v0.11.0'
```

Comment:
Theres a community action (https://github.com/typst-community/setup-typst) that we should use instead